packages/shared/lib/core/router/app-router.ts

```diff
@@ -128,11 +109,7 @@ export class AppRouter extends Router<AppRoute> {
                 const { importType } = params
                 walletSetupType.set(importType as unknown as SetupType)
                 nextRoute = AppRoute.Congratulations
-                if (importType === ImportType.Mnemonic) {
-                    nextRoute = AppRoute.Secure
-                } else if (
-                    [ImportType.Stronghold, ImportType.TrinityLedger, ImportType.FireflyLedger].includes(importType)
-                ) {
+                if ([ImportType.Stronghold, ImportType.TrinityLedger, ImportType.FireflyLedger].includes(importType)) {
```

Comment:
`if (importType === ImportType.Mnemonic) {`
Will it not be possible to migrate a profile with mnemonic from `Chrysalis` to `Stardust`?
I think this is done directly in `Firefly`, so maybe it's not necessary here

Reply:
the task says: `Allow only migrating using 81 characters, a seedvault file (needs file validation), or a trinity ledger backup` that's why i remove this :see_no_evil: